spec/components/parameters/startDate.ts

```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../utils/ref';
+
+const startDate: ParameterObject = {
+  name: 'startDate',
```

Comment:
In order to respect the pattern used in the previous APIs, in my opinion, would be better to call this parameter as 'startAt'.